mimesniff/mime-types/resources/mime-types.json

```diff
@@ -193,6 +193,23 @@
     "input": "x/x;test=\"\\",
     "output": "x/x;test=\"\\\\\""
   },
+  "Whitespace (not handled by generated-mime-types.json)",
+  {
+    "input": "x/x;x= ",
+    "output": "x/x"
+  },
+  {
+    "input": "x/x;\" ",
+    "output": "x/x;\" \""
```

Comment:
In my parser the output is `x/x`, with no parameters. There is no parameter value here, so this is expected.